puppet/modules/jenkins_master/manifests/init.pp

```diff
@@ -0,0 +1,197 @@
+class jenkins_master {
+
+  jenkins::config { 'config_hash':
+    'JENKINS_JAVA_OPTIONS' => {
+      'value' => '-Djava.awt.headless=true -Djenkins.install.runSetupWizard=false -Xms2048m -Xmx2048m',
```

Comment:
iirc this is coming right now from foreman (ENC)

Reply:
Yes it is.

My goal is to get as much of our Jenkins into source control and code as possible to ease rebuilding if necessary in the future. I want our Jenkins to be [cattle instead of a pet](https://www.engineyard.com/blog/pets-vs-cattle)